Contribute/dotnet/labels-projects.md

```diff
@@ -1,16 +1,17 @@
 ---
-title: Labels and projects roadmap 
-description: This article explains how labels and projects are used in the dotnet/docs repository.
+title: Labels, projects, and milestones roadmap
+description: This article explains how labels, projects, and milestones are used in the dotnet/docs repository.
 ms.topic: contributor-guide
 ms.prod: non-product-specific
 ms.custom: external-contributor-guide
-ms.date: 03/24/2020
+ms.date: 08/06/2020
 ---
-# Labels and projects roadmap
 
-The .NET docs team makes extensive use of [GitHub labels](https://github.com/dotnet/docs/labels) to organize our work. By filtering on combinations of labels, we can quickly focus on sections of interest on the [.NET docs website](https://docs.microsoft.com/dotnet).
+# Labels, projects, and milestones roadmap
 
-We also use [GitHub projects](https://github.com/dotnet/docs/projects) to organize sprints and other goal-oriented epics.
+The .NET docs team makes extensive use of [GitHub labels](https://github.com/dotnet/docs/labels) to organize our work. By filtering on combinations of labels, we can quickly focus on sections of interest on the [.NET docs website](https://docs.microsoft.com/dotnet). For example, we could filter to all of the open priority one `P1` issues with a query to [`is:open label:p1 repo:dotnet/docs`](https://github.com/pulls?q=is%3Aopen+label%3Ap1+repo%3Adotnet%2Fdocs).
+
+We use [GitHub projects](https://github.com/dotnet/docs/projects) to organize sprints and other goal-oriented epics. We also use [GitHub milestones](https://github.com/dotnet/docs/milestones) to track work. It is best to think of projects for planning (issues), and milestones for work (pull requests).
```

Comment:
If we're going to mention milestones we should explain exactly how we use them.  I'm inclined to think of them as not important enough to mention here.

Reply:
The entire idea behind this update to the article was to add how milestones relate to our work. It was something that was unclear to me, and we need to be explicit about them.

Reply:
I'm fine with including mention of milestones, but if we mention them we should explain how and why we use them.